server/services/v1/database/search_runner.go

```diff
@@ -63,6 +65,14 @@ func (runner *SearchQueryRunner) ReadOnly(ctx context.Context, tenant *metadata.
 	if err != nil {
 		return Response{}, ctx, err
 	}
+	if config.DefaultConfig.Search.LogFilter {
+		log.Error().
```

Comment:
Wouldn't this be info log?

Reply:
I wish Go's logging libraries were as mature as Java ;) 's logging libraries.

Reply:
The log level is currently set as `Error` to reduce the info logs. We need to change that to Warn and move logs like these to War as the info needs more cleanup.